airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -1,18 +1,19 @@
 """Prometheus exporter for Airflow."""
+import datetime
 import json
 import os
 import pickle
+import pytz
 from contextlib import contextmanager
-from datetime import datetime
 
-import croniter
 import dateparser
 import pendulum
 from flask import Response
 from flask_admin import BaseView, expose
 from prometheus_client import REGISTRY, generate_latest
 from prometheus_client.core import GaugeMetricFamily
-from sqlalchemy import Column, Float, String, and_, func
+from pytimeparse import parse
```

Comment:
when I use thees generic parse functions, I like to rename them
e.g
from pytimeparse import parse as time_parse
from dateparser import parser as date_parser

Reply:
Let me modify it.